Cinema system/Cinema system/Models/CinemaDBContext.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Cinema_system.Models
+{
+    public partial class CinemaDBContext : DbContext
+    {
+        public CinemaDBContext()
+        {
+        }
+
+        public CinemaDBContext(DbContextOptions<CinemaDBContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<Cinema> Cinema { get; set; }
+        public virtual DbSet<Hall> Hall { get; set; }
+        public virtual DbSet<Movie> Movie { get; set; }
+        public virtual DbSet<Order> Order { get; set; }
+        public virtual DbSet<Seance> Seance { get; set; }
+        public virtual DbSet<SeanceSeats> SeanceSeats { get; set; }
+        public virtual DbSet<Seat> Seat { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
+                optionsBuilder.UseSqlServer("Data Source=LAPTOP-HTQV3HQS\\MSSQLSERVER01;Initial Catalog=CinemaDB;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False");
```

Comment:
Move connection string to appsettings.json. Use standard configuration tools to access it on application startup.